includes/Core/Util/Activation.php

```diff
@@ -186,21 +258,52 @@ private function get_activation_notice() {
 											); // phpcs:ignore WordPress.XSS.EscapeOutput.OutputNotEscaped
 											?>
 										</div>
-										<h3 class="googlesitekit-heading-1 googlesitekit-activation__title">
+										<h3 class="googlesitekit-heading-3 googlesitekit-activation__title">
 											<?php esc_html_e( 'Congratulations, the Site Kit plugin is now activated.', 'google-site-kit' ); ?>
 										</h3>
 									</div>
 									<div class="
 										mdc-layout-grid__cell
 										mdc-layout-grid__cell--start-8-desktop
 										mdc-layout-grid__cell--offset-1-desktop
-										mdc-layout-grid__cell--align-middle
+										mdc-layout-grid__cell--align-bottom
 									">
-										<a href="#" onClick="javascript:if( 'undefined' !== typeof sendAnalyticsTrackingEvent ) { sendAnalyticsTrackingEvent( 'plugin_setup', 'goto_sitekit' ) };document.location='<?php echo esc_url( $sitekit_splash_url ); ?>';"
-											class="googlesitekit-activation__button mdc-button mdc-button--raised">
+										<a id="start-setup-link" href="<?php echo esc_url( $sitekit_splash_url ); ?>" class="googlesitekit-activation__button mdc-button mdc-button--raised">
 											<?php esc_html_e( 'Start Setup', 'google-site-kit' ); ?>
 										</a>
 									</div>
+									<div class="
+										mdc-layout-grid__cell
+										mdc-layout-grid__cell--span-12
+									">
+										<div class="googlesitekit-opt-in">
+											<div class="mdc-form-field">
+												<div class="mdc-checkbox mdc-checkbox--upgraded mdc-ripple-upgraded mdc-ripple-upgraded--unbounded">
+													<input class="mdc-checkbox__native-control" type="checkbox" id="opt-in" name="optin" value="1" />
+													<div class="mdc-checkbox__background">
+														<svg class="mdc-checkbox__checkmark" viewBox="0 0 24 24">
+															<path class="mdc-checkbox__checkmark-path" fill="none" d="M1.73,12.91 8.1,19.28 22.79,4.59"></path>
+														</svg>
+														<div class="mdc-checkbox__mixedmark">
+														</div>
+													</div>
+												</div>
+												<label for="opt-in">
+													<?php
+														echo sprintf(
+															// translators: Placeholders here are for <a> + </a> tags.
+															esc_html__(
+																'Help us improve the Site Kit plugin by allowing tracking of anonymous usage stats. All data are treated in accordance with %1$sGoogle Privacy Policy%2$s.',
+																'google-site-kit'
+															),
+															'<a href="https://policies.google.com/privacy" rel="noopener noreferrer">',
+															'</a>'
```

Comment:
The anchor tag should be within the translatable string, only the `href` value should be inserted with `sprintf`.
The privacy policy URL should also be translatable/localized.

You can see an example from this in core here https://github.com/WordPress/WordPress/blob/8662d9a74d67669da86e2f5181f1f972229648e8/wp-admin/admin-footer.php#L35-L39

For escaping, we can use `wp_kses` instead of `esc_html` which I'm guessing is why you separated the two 😄

Reply:
Hmm, but why? There's no reason to translate the `<a href="">` tag itself and allowing it to be translated introduces a possible issue if the translator messes up the HTML.

Translating the privacy policy link is a good call though 👍. I'll add that as a third string.

Reply:
(I separated them because the `<a>` tags don't need to be translated, not for any other reason.)

Reply:
I've added the localisation to the privacy policy link, but I've kept the `<a>` tag out of the translation string—this reduces potential translator error but also reduces the size of each translation bundle. I don't see a reason to allow translators to change the contents of the `<a>` tag as they can still move the `%$1s` and `%$2s` around within the translation string if needed.

Reply:
> There's no reason to translate the `<a href="">` tag itself and allowing it to be translated introduces a possible issue if the translator messes up the HTML.

It's true that the html itself does not require translation, but extracting the html to placeholders is also possible to be translated incorrectly (e.g. with an RTL language, where html placeholders still need to be LTR). It makes for a simpler translation with less room for error with a single placeholder for the href. We can assume that translators can handle minimal basic markup. We should aim to minimize any markup in translated strings, but let's not avoid it unless necessary. In this case I think we should follow the more common convention.

Reply:
@tofumatt I agree with @aaemnnosttv, the convention is to include tags in scenarios like that. For translators, something like `%1$sanchor text%2$s` is harder to understand than `<a href="%1$s">anchor text</a>`, and the latter is also the recommendation from the WordPress translation teams.

Reply:
Fair enough! We always took a different approach at Mozilla, and most of my Gutenberg i18n stuff didn't involve markup. I've make the change 👍